modules/assim.batch/R/pda.emulator.R

```diff
@@ -145,9 +145,8 @@ pda.emulator <- function(settings, params.id=NULL, param.names=NULL, prior.id=NU
   }
   
   # define range to make sure mcmc.GP doesn't propose new values outside 
-  rng=matrix(c(sapply(prior.fn$qprior[prior.ind] ,eval,list(p=0)),
-        sapply(prior.fn$qprior[prior.ind] ,eval,list(p=1))),
-        nrow=n.param)
+  
+  rng=matrix(c(apply(X,2,min), apply(X ,2,max)),nrow=n.param)
```

Comment:
@mdietze I thought with the emulator the proposed values should not only stay within the prior range but also within the range of design points as the behaviour of the emulator at the edges can be unpredictable (e.g. regressing towards the mean), but I can change it back if we need to think about it